### PR TITLE
refresh token method

### PR DESCRIPTION
**Pull request by hkd987**
simple way to handle the refresh token.

### CODE REVIEW

**Comment by hkd987** [comment]
ENJOY!!!

File: src/VueGAPI/GoogleAuthService.js [hunk 1]
```diff
@@ -5,16 +5,34 @@ export default class GoogleAuthService {
...
+    return JSON.stringify(authResult.expires_in * 1000 + new Date().getTime())
+  }
+
   login (event) {
     return this.authInstance.signIn()
       .then(this.setSession)
   }
 
+  refreshToken (event) {
+  console.log('REFRESHED TOKEN')
```
Comment (rcastera) [inline src/VueGAPI/GoogleAuthService.js]: @	hkd987 you might want to remove this.

Comment (cedpoilly) [reply src/VueGAPI/GoogleAuthService.js]: @hkd987 do you still need this log yet? Else let's remove it before the merge :)
```diff
-  console.log('REFRESHED TOKEN')
```

File: README.md [hunk 2]
```diff
@@ -113,3 +115,17 @@ export default {
...
+
+## refreshToken
+
+This will shortcut getting a refresh token from Google, this should be placed in your App.vue on the created page and run on a timer of 45min
+```html
+<script>
+  name: 'App'
+
+  created () {
+  // NOTE: this throws errors when the user is signed out....
```
Comment (cedpoilly) [inline README.md]: Is there a way to catch those errors?

File: README.md [hunk 3]
```diff
@@ -113,3 +115,17 @@ export default {
...
+  window.setInterval(this.$refreshToken(), 2.7e+6)
```
Comment (cedpoilly) [inline README.md]: @hkd987 what's the purpose of the interval & it's huge (`2.7e+6`) delay?

File: src/VueGAPI/GoogleAuthService.js (hunk 1 above)
Comment (hkd987) [reply src/VueGAPI/GoogleAuthService.js]: I let it there to be similar to what the style you had going where you are logging when GAPI is installed. We can remove it if you want.

File: README.md (hunk 2 above)
Comment (hkd987) [reply README.md]: Not sure....maybe just wrap it in a try catch would be the simplest way. My thought was that the user of the plugin could do that bit as it’s not directly related to the plugin.

File: README.md (hunk 3 above)
Comment (hkd987) [reply README.md]: That is 45min. That is the recommended time that google ask you to refresh. The token is dead after one hour, and google says not to refresh more than every 45min.

**Comment by hkd987** [comment]
made some changes we discussed and issues that came up as I fixed those issues

**Comment by cedpoilly** [comment]
Thanks. Will test & merge soon

**Comment by rcastera** [comment]
@cedpoilly have you bumped the version?

**Comment by rcastera** [comment]
Build fails on my side as well with:

```
> vue-gapi@0.0.8 build /Users/richardcastera/Sites/vue-gapi
> node build/build.js

{ Error: Unexpected token (15:13) in /Users/richardcastera/Sites/vue-gapi/src/VueGAPI/GoogleAuthService.js
    at error (/Users/richardcastera/Sites/vue-gapi/node_modules/rollup/dist/rollup.js:170:12)
    at /Users/richardcastera/Sites/vue-gapi/node_modules/rollup/dist/rollup.js:8937:6
  pos: 450,
  loc: Position { line: 15, column: 13 },
  raisedAt: 451,
  plugin: 'commonjs',
  id: '/Users/richardcastera/Sites/vue-gapi/src/VueGAPI/GoogleAuthService.js' }
{ Error: Unexpected token (15:13) in /Users/richardcastera/Sites/vue-gapi/src/VueGAPI/GoogleAuthService.js
    at error (/Users/richardcastera/Sites/vue-gapi/node_modules/rollup/dist/rollup.js:170:12)
    at /Users/richardcastera/Sites/vue-gapi/node_modules/rollup/dist/rollup.js:8937:6
  pos: 450,
  loc: Position { line: 15, column: 13 },
  raisedAt: 451,
  plugin: 'commonjs',
  id: '/Users/richardcastera/Sites/vue-gapi/src/VueGAPI/GoogleAuthService.js' }
{ Error: Unexpected token (15:13) in /Users/richardcastera/Sites/vue-gapi/src/VueGAPI/GoogleAuthService.js
    at error (/Users/richardcastera/Sites/vue-gapi/node_modules/rollup/dist/rollup.js:170:12)
    at /Users/richardcastera/Sites/vue-gapi/node_modules/rollup/dist/rollup.js:8937:6
  pos: 450,
  loc: Position { line: 15, column: 13 },
  raisedAt: 451,
  plugin: 'commonjs',
  id: '/Users/richardcastera/Sites/vue-gapi/src/VueGAPI/GoogleAuthService.js' }
{ Error: Unexpected token (15:13) in /Users/richardcastera/Sites/vue-gapi/src/VueGAPI/GoogleAuthService.js
    at error (/Users/richardcastera/Sites/vue-gapi/node_modules/rollup/dist/rollup.js:170:12)
    at /Users/richardcastera/Sites/vue-gapi/node_modules/rollup/dist/rollup.js:8937:6
  pos: 450,
  loc: Position { line: 15, column: 13 },
  raisedAt: 451,
  plugin: 'commonjs',
  id: '/Users/richardcastera/Sites/vue-gapi/src/VueGAPI/GoogleAuthService.js' }
```

**Comment by cedpoilly** [comment]
@rcastera @hkd987  kindly check version 0.0.10 on npm :)
I'll close this issue once I have both confirmation ;)
Thank you both for your contribution

**Comment by hkd987** [comment]
Version on npm is 0.0.10

**Pull request merged**